publication-commons/src/main/java/no/unit/nva/publication/service/impl/UpdateResourceService.java

```diff
@@ -163,85 +173,6 @@ private TransactWriteItem updateResource(Resource resourceUpdate) {
-                                        + "#PK1 = :PK1, "
-                                        + "#SK1 = :SK1 ";
-        
-        final String conditionExpression = "#data.#status <> :publishedStatus";
-        
-        Map<String, String> expressionNamesMap = Map.of(
-            "#data", RESOURCE_FIELD_IN_RESOURCE_DAO,
-            "#status", STATUS_FIELD_IN_RESOURCE,
-            "#modifiedDate", MODIFIED_FIELD_IN_RESOURCE,
-            "#publishedDate", PUBLISHED_DATE_FIELD_IN_RESOURCE,
-            "#PK1", DatabaseConstants.BY_TYPE_CUSTOMER_STATUS_INDEX_PARTITION_KEY_NAME,
-            "#SK1", DatabaseConstants.BY_TYPE_CUSTOMER_STATUS_INDEX_SORT_KEY_NAME
-        );
-        
-        Map<String, AttributeValue> expressionValuesMap = Map.of(
-            ":newStatus", new AttributeValue(PublicationStatus.PUBLISHED.getValue()),
-            ":modifiedDate", new AttributeValue(nowString),
-            ":publishedStatus", new AttributeValue(PublicationStatus.PUBLISHED.getValue()),
-            ":PK1", new AttributeValue(dao.getByTypeCustomerStatusPartitionKey()),
-            ":SK1", new AttributeValue(dao.getByTypeCustomerStatusSortKey()));
-    
-        Update update = new Update()
-                            .withTableName(tableName)
-                            .withKey(dao.primaryKey())
-                            .withUpdateExpression(updateExpression)
-                            .withConditionExpression(conditionExpression)
-                            .withExpressionAttributeNames(expressionNamesMap)
-                            .withExpressionAttributeValues(expressionValuesMap);
-        return new TransactWriteItem().withUpdate(update);
-    }
-    
-    private TransactWriteItem updateDoiRequestResourceStatus(DoiRequestDao doiRequestDao, String nowString) {
-        
-        String updateExpression = "SET "
-                                  + "#data.#resourceStatus = :publishedStatus,"
-                                  + "#data.#modifiedDate = :modifiedDate ";
-        Map<String, String> expressionAttributeNames = Map.of(
-            "#data", DOI_REQUEST_FIELD_IN_DOI_REQUEST_DAO,
-            "#resourceStatus", RESOURCE_STATUS_FIELD_IN_DOI_REQUEST,
-            "#modifiedDate", MODIFIED_DATE_FIELD_IN_DOI_REQUEST
-        );
-        Map<String, AttributeValue> attributeValueMap = Map.of(
-            ":publishedStatus", new AttributeValue(PublicationStatus.PUBLISHED.getValue()),
-            ":modifiedDate", new AttributeValue(nowString)
-        );
-    
-        Update update = new Update().withTableName(tableName)
-                            .withKey(doiRequestDao.primaryKey())
-                            .withUpdateExpression(updateExpression)
-                            .withExpressionAttributeNames(expressionAttributeNames)
-                            .withExpressionAttributeValues(attributeValueMap);
-        
-        return new TransactWriteItem().withUpdate(update);
-    }
-    
```

Comment:
This code is now obsolete, because we do a lot of things when we publish a publication and it is easier to use the updatePublication method.